docs/hardware.rst

```diff
@@ -649,3 +649,30 @@ servers, since you will need to connect them to a monitor to do the
 initial Ubuntu installation.
 
 You should use a wired (USB) keyboard and mouse, not wireless.
+
+
+Hardware End-of-Life
+^^^^^^^^^^^^^^^^^^^^
+
+No matter what hardware you decide to use, it's important to be mindful of
+how long it will continue to receive security updates. Given the security
+requirements for a SecureDrop instance, any hardware that is no longer
+receiving security updates from the manufacturer will become more and more
+vulnerable over time. Once your hardware has reached its end-of-life (EOL),
+we recommend upgrading to newer, supported hardware.
+
+For the hardware we recommend, you can find a list of end-of-life dates below:
+
+================  ====================================================
+Hardware          End-of-Life (EOL)                                       
+================  ====================================================
+Intel NUC11PAHi3  June 30, 2025                                       
+Intel NUC10i5FNH  June 25, 2024                                       
+Intel NUC8i5BEK   March 26, 2024                                      
+Intel NUC7i5BNH   April 30, 2023                                      
+Thinkpad T420     Already EOL; last BIOS update 2018                  
+Thinkpad T440     No official EOL; last BIOS update 2020              
```

Comment:
It might make sense to update our recommendations for the Journalist/Admin Workstation to refer to the Thinkpad T Series more generally, as part of this change, so we're not recommending models that are already EOL. (For the SVS the remaining reference to the T420 may be more defensible, given that we're also suggesting considering older hardware for the air-gap.)

Reply:
Those are great points, thanks! I've just made those changes and committed them here. When you have a moment, please review and let me know if you like the way I implemented these suggestions before I mark this as resolved. Thanks!

Reply:
Looks great! I think we can also safely remove the clause "and have had good experiences specifically with the T420 and T440" in the Journalist/Admin Workstation recommendation section, since both are EOL.